client/my-sites/customer-home/style.scss

```diff
@@ -58,6 +58,10 @@
 		@include grid-template-columns( 12, 24px, 1fr );
 		grid-gap: 24px;
 	}
+
+	h2 {
+		font-family: 'Recoleta', serif;
```

Comment:
We'll need to download the font, we can maybe check in to see how Gutenboarding is doing it. There are some performance considerations here as we do it, so we can maybe consult perfops on best practices.